README.md

```diff
@@ -1,2 +1,96 @@
-# uniforms-ionic
-Uniforms ionic
+# Basic usage
+
+### 1. Install the required packages
+
+To start using uniforms, we have to install three independent packages:
+
+1. Core
+2. Bridge
+3. Theme
+
+In this example, we will use the JSONSchema to describe our desired data format and style our form using Semantic UI theme.
+
+```shell
+npm install uniforms@3.0.0-alpha.4
+npm install uniforms-bridge-simple-schema2@3.0.0-alpha.4
+npm install simpl-schema
+npm install uniforms-ionic
+```
+
+<!-- **Note**: When using a themed package, remember to include correct styles! If you are willing to run this example by yourself,
+have a read on [Semantic UI React's theme usage](https://react.semantic-ui.com/usage/#theme). -->
+
+### 2. Start by defining a schema
+
+After we've installed required packages, it's time to define our Guest schema. We can do it in a plain JSON, which is a valid JSONSchema instance:
+
+```javascript
+import SimpleSchema from 'simpl-schema';
+
+const schema = new SimpleSchema({
+  name: {
+    type: String
+  },
+  lastname: {
+    type: String
+  },
+  date: {
+    type: Date
+  }
+});
+```
+
+### 3. Then create the bridge
+
+Now that we have the schema, we can create the uniforms bridge of it, by using the corresponding uniforms schema-to-bridge package.
+Creating the bridge instance is necessary - without it, uniforms would not be able to process form generation and validation.
+As we are using the SimplSchema, we have to import the `uniforms-bridge-simple-schema2` package.
+
+```js
+import { SimpleSchema2Bridge } from 'uniforms-bridge-simple-schema-2';
+
+...
+
+export default new SimpleSchema2Bridge(schema);
+```
+
+Just to recap, the whole `schema.js` file looks like this:
+
+```js
+import SimpleSchema from 'simpl-schema';
+import { SimpleSchema2Bridge } from 'uniforms-bridge-simple-schema-2';
+
+const schema = new SimpleSchema({
+  name: {
+    type: String
+  },
+  lastname: {
+    type: String
+  },
+  date: {
+    type: Date
+  }
+});
+
+export default new SimpleSchema2Bridge(schema);
+```
+
+### 4. Finally, use it in a form!
+
+Uniforms theme packages provide the `AutoForm` component, which is able to generate the form based on the given schema.
+All we have to do now is to pass the previously created GuestSchema to the `AutoForm`:
+
+```js
+import React from 'react';
+import { AutoForm } from uniforms-semantic;
+
+import schema from './schema';
+
+export default function GuestForm() {
+  return <AutoForm schema={schema} onSubmit={console.log} />;
+}
+```
+
+And that's it! `AutoForm` will generate a complete form with labeled fields, errors list (if any) and a submit button.
```

Comment:
I think we need to as follow up
Copy uniforms register here or.. include it into our packace

Reply:
Yeah I need to update the repo a little bit. Will try do it over the next couple of days. Need to sort out a ui issue with the patternfly repo too.